apps/base-docs/tutorials/docs/2_data-indexers-subquery.md

```diff
@@ -0,0 +1,373 @@
+---
+title: 'Indexing Data with SubQuery'
+slug: /subquery
```

Comment:
Sorry, my other comment was meant to be change `/subquery` to `/data-indexers-subquery`.